backend/utilities/generalUtils.ts

```diff
@@ -19,3 +19,11 @@ export const calculateMedianScore = (values: number[]): number => {
   }
   return (values[mid] + values[mid - 1]) / 2;
 };
+
+export const isCompletedTestSession = (
+  results: Array<Array<Array<number>>>,
+): boolean => {
+  return results.every((question) =>
+    question.every((subquestion) => subquestion.length !== 0),
+  );
```

Comment:
```suggestion
  results.flat().every((answer) => answer.length);
```
might be a little easier to read?